emscripten.py

```diff
@@ -1411,7 +1411,7 @@ def asmjs_mangle(name):
         metadata['declares'].append(func_name)
     elif line.startswith('  (func '):
       parts = line.split(' ')
```

Comment:
Would making this just be `line.split()` work here instead of stripping in the next line?